build/Dockerfile

```diff
@@ -1,6 +1,6 @@
 #################################################################################
 # Copyright (c) 2022,2023 T-Systems International GmbH
-# Copyright (c) 2022,2023 Contributors to the Eclipse Foundation
+# Copyright (c) 2022,2023,2024 Contributors to the Eclipse Foundation
```

Comment:
Please follow [TRG 7.02](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) with regards to years range. In this example should be "2022, 2024" which determines range from 2022 to 2024.

Reply:
this has been resolved